i18n/changed.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+"""
+Determine if the source translation files are up-to-date.
+"""
+from __future__ import print_function
+from subprocess import CalledProcessError
+import sys
+
+from i18n import Runner
+from i18n.execute import execute
+
+
+class Changed(Runner):
+    """
+    Class used to check if the source translation files are up-to-date
+    """
+    def run(self, args):
+        """
+        Main entry point of script
+        """
+        changes_detected = self.detect_changes()
+        message = self.get_message(changes_detected)
+        print(message)
+        sys.exit(1)
```

Comment:
This is not the right way to return a status code:
1) Most importantly, it is always 1, when it should depend on whether changes were detected or not.
2) This method should return the status code, though you have to fix i18n.__init__.Runner.__call__ to return it also.